sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -141,26 +141,21 @@ To begin translating your documents, pass a list of `DocumentTranslationInput` i
 Constructing a `DocumentTranslationInput` requires that you pass the SAS URLs to your source and target containers (or files)
 and the target language(s) for translation.
 
-A single source container with documents can be translated to many different languages:
+A single source container with documents can be translated to a different language:
 
 ```python
-from azure.ai.translation.document import DocumentTranslationInput, TranslationTarget
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.translation.document import DocumentTranslationClient
 
-my_input = [
-    DocumentTranslationInput(
-        source_url="<sas_url_to_source>",
-        targets=[
-            TranslationTarget(target_url="<sas_url_to_target_fr>", language_code="fr"),
-            TranslationTarget(target_url="<sas_url_to_target_de>", language_code="de")
-        ]
-    )
-]
+document_translation_client = DocumentTranslationClient("<endpoint>", AzureKeyCredential("<api_key>"))
+poller = document_translation_client.begin_translation("<sas_url_to_source>", "<sas_url_to_target", "fr")
```

Comment:
nit: do you think it is worth creating a variable  `language_code="fr"` and then use that variable so the user understands what `fr` is? or maybe just follow the same pattern you have and do `<language_code>`

Reply:
saw we do this in .NET too. If you agree I can create an issue for .NET. If you think it is not important will leave as is :)

Reply:
I was trying to demonstrate that these can be passed positionally. I think I'll update to "<target_language_code>" here to be consistent. Thanks for pointing that out!